pyproject.toml

```diff
@@ -29,7 +29,8 @@ classifiers = [
 ]
 dependencies = [
     "certifi",
-    "httpcore>=0.18.0,<0.19.0",
+    "httpcore",
```

Comment:
Don't we need to pin version for httpcore and anyio here?

Reply:
If we want pinning here then...

* We should probably pin anyio to the `>=4.0.0,<5.0.0` range.
* We can pin httpcore to the `>=1.0.0,<2.0.0` range. (Actually I think we can go lower than that, but...)